fieldservice/models/fsm_order.py

```diff
@@ -371,3 +368,20 @@ def _onchange_template_id(self):
                 self.type = self.template_id.type_id
             if self.template_id.team_id:
                 self.team_id = self.template_id.team_id
+
+    def _get_location_directions(self, location_id):
+        self.location_directions = ""
+        s = self.location_id.direction
+        parent_location = self.location_id.fsm_parent_id
+        # ps => Parent Location Directions
+        # s => String to Return
+        while parent_location.id is not False:
+            import pdb; pdb.set_trace()
+            ps = parent_location.direction
+            if ps:
+                if s:
+                    s = s + parent_location.direction
+                else:
+                    s = parent_location.direction
```

Comment:
@osi-scampbell What about?
```suggestion
                s += parent_location.direction
```

Reply:
Need the if/else to check if s is False or else it throws a TypeMismatch error for 'NoneType' and 'str'

Reply:
@osi-scampbell You can initialize `s` with:

```
s = self.location_id.direction or ""
```